exercises/build.gradle

```diff
@@ -9,14 +9,14 @@ subprojects { project ->
       java.srcDirs = ["src/example/java"]
     }
     project["compileJava"].doFirst { compileTask ->
-      println "  (source = " + compileTask.source.asPath + ")"
+      println "  (source = ${compileTask.source.asPath})"
     }
 
     starterSource {
       java.srcDirs = ["src/main/java"]
     }
     project["compileStarterSourceJava"].doFirst { compileTask ->
-      println "  (source = " + compileTask.source.asPath + ")"
+      println "  (source = ${compileTask.source.asPath})"
```

Comment:
From the documentation it looks like you can also use just `$compileTask.source.asPath`? But I'm happy to keep the curly brackets if leaving them out doesn't work or you think it's clearer to keep them :)